scispacy/candidate_generation.py

```diff
@@ -109,39 +131,54 @@ class CandidateGenerator:
     ann_concept_aliases_list: List[str]
         A list of strings, mapping the indices used in the ann_index to possible UMLS mentions.
         This is essentially used a lookup between the ann index and actual mention strings.
-    umls: UmlsKnowledgeBase
```

Comment:
should this still be called umls? or something more generic?

Reply:
yeahhh this is kind of annoying. I could change it to `kb` everywhere, leaving `umls` there also for a couple of releases or something?

Reply:
You have more experience with changes like this from allennlp, so happy to defer to you, but that sounds reasonable to me. add a deprecation warning when using umls or something maybe

Reply:
cool, will do